.github/workflows/ci.yml

```diff
@@ -4,15 +4,15 @@ on:
   push:
     branches:
       - master
-      - 2201.[0-9]+.x
+      - 2201.10.x
```

Comment:
You can revert this back. I think `2201.[0-9]+.x` is added to run it for all the 2201.xx.x branches

Reply:
Sure. Changed with [895c030](https://github.com/ballerina-platform/module-ballerinax-salesforce/pull/371/commits/895c0300c09e630b2a6c1106e78da1f2d6643d6a).